cmd/clusterctl/cmd/root.go

```diff
@@ -20,45 +20,100 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"strings"
 
+	"github.com/MakeNowJust/heredoc"
```

Comment:
Do we have to worry about license approval for this? I'm not sure what the exact policy is for k8s projects and OSS licenses.

Reply:
I will ask around, but this is used also in K/K -more specifically in clusterctl-, so I guess it is ok

Reply:
It is MIT license, so it should not be a problem